var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -432,6 +434,8 @@ def configure_args(self):
 
         config_args += self.enable_or_disable("fsync")
 
+        config_args.extend(self.enable_or_disable("logging"))
```

Comment:
This is nitpicking but shall we keep the code homogeneous as in my previous request?

```suggestion
        config_args += self.enable_or_disable("logging")
```

@alalazo please, feel free to merge this as-is if you think that I'm asking for too much :)